tests/test_core.py

```diff
@@ -9267,10 +9267,10 @@ def setUp(self):
 corez = make_run('corez', emcc_args=['-Oz'])
 
 # MEMORY64=1
-wasm64 = make_run('wasm64', emcc_args=[], settings={'MEMORY64': 1},
+wasm64 = make_run('wasm64', emcc_args=['--profiling-funcs'], settings={'MEMORY64': 1},
                   require_v8=True, v8_args=['--experimental-wasm-memory64'])
 # MEMORY64=2, or "lowered"
-wasm64l = make_run('wasm64l', emcc_args=[], settings={'MEMORY64': 2},
+wasm64l = make_run('wasm64l', emcc_args=['--profiling-funcs'], settings={'MEMORY64': 2},
```

Comment:
maybe worth a TODO to remove the profiling-funcs when stable?

Reply:
I'm actually a fan of figuring out way to get function names on by default for most/some/all test configurations.   Its so annoying to see failures in CI where the backtraces have no symbol information at all.   I know that argument that we should test the default by default, but its may a place where we can make an exception, at least the majority of the test configurations.

We discussed this already in https://github.com/emscripten-core/emscripten/pull/15480 but I'd like to continue to push for it in some form.

Reply:
Added a comment